codesniff/app/views.py

```diff
@@ -108,7 +108,7 @@ def get_queryset(self):
         if language is not None:
             queryset = queryset.filter(language=language)
         if username is not None:
-            queryset = queryset.filter(creator__username=username)
+            queryset = queryset.filter(creator=username)
```

Comment:
@hxkoh2 I changed this because we were calling it creator in the backend, not creator__username. It didn't seem to be used anywhere else. Can you confirm? Thanks!